e2e/test-01-backup.bats

```diff
@@ -12,10 +12,13 @@ DETIK_CLIENT_NAMESPACE="k8up-e2e-subject"
 DEBUG_DETIK="true"
 
 @test "verify a backup" {
+	expected_content="expected content: $(timestamp)"
+	expected_filename="expected_filename.txt"
+
 	given_a_running_operator
 	given_a_clean_ns
 	given_s3_storage
-	given_a_subject
+	given_a_subject "${expected_filename}" "${expected_content}"
 
 	apply definitions/backup
 	try "at most 10 times every 1s to get backup named 'k8up-k8up-backup' and verify that '.status.started' is 'true'"
```

Comment:
Line 26 (can't select that in GH diff) with `try "at most 10 times every 2s to find 1 pod named 'k8up-k8up-backup-.....' with 'status' being 'Running'"` sometimes fails if the pod is `Succeeded` before the `try` is querying the status. I would just remove it and rely on the next assertion.